spring-boot-project/spring-boot-docs/src/main/java/org/springframework/boot/docs/service/KotlinDatabaseAccountService.kt

```diff
@@ -0,0 +1,18 @@
+package org.springframework.boot.docs.service
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+/**
+ * * Sample showing autowiring the [org.springframework.beans.factory.annotation.Autowired] annotation
+ */
+// tag::customizer[]
+@Service
+class KotlinDatabaseAccountService(@Autowired private val riskAssessor: RiskAssessor) : AccountService {
```

Comment:
The be equivalent to the Java version, `@Autowired` should be on the constructor:
`class KotlinDatabaseAccountService @Autowired contructor(private val riskAssessor: RiskAssessor) : AccountService {`